src/input/source/hidraw/steam_deck.rs

```diff
@@ -17,14 +17,18 @@ use tokio::sync::{
 use crate::{
     drivers::{
         dualsense::hid_report::SetStatePackedOutputData,
+        lego::driver,
```

Comment:
Why are we importing the lego driver in the steam deck driver? o.O

Reply:
Automatic import. Good catch